SCAutolib/utils.py

```diff
@@ -126,3 +128,25 @@ def dump_to_json(obj):
     with obj.dump_file.open("w") as f:
         json.dump(obj.__dict__, f)
     logger.debug(f"Object {type(obj)} is stored to the {obj.dump_file} file")
+
+
+def user_factory(username):
+    """
+    Load user with given username from JSON file. If user have the card file
+    linked, then load it as well.
+
+    :param username: username of the user
+    :type username: str
+
+    :return: user object
+    :rtype: BaseUser
+    """
+    user_file = LIB_DUMP_USERS.joinpath(f"{username}.json")
```

Comment:
What is the logic behind this `LIB_DUMP_USERS` directory? Meaning from user's perspective. Do I set this variable at some point? Is it pointing to a directory where data is automatically dumped to be later used?

Reply:
So this function is only for creating user objects for users that were previously explicitly dumped? I might not understand the full picture still..

Reply:
Yes, exactly as you said